projects/aws/eks-a-snow-admin-ami/Makefile

```diff
@@ -0,0 +1,59 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG=0.0.0-dev # This value does not matter but needs to be set
+
+REPO=eks-a-snow-admin-ami
+REPO_OWNER=aws
+
+BUILD_TARGETS=validate
+RELEASE_TARGETS=build-raw-image
+
+PACKER_VERSION=1.8.0
+
+EKSA_VERSION?=v0.8.0
+EKSA_RELEASE_MANIFEST_URL?=https://anywhere-assets.eks.amazonaws.com/releases/eks-a/manifest.yaml
+
+ARTIFACTS_UPLOAD_PATH?=$(PROJECT_PATH)
+EXPORT_AMI_BUCKET?=$(ARTIFACTS_BUCKET)
+FULL_OUTPUT_DIR=$(MAKE_ROOT)/$(OUTPUT_DIR)/ami/snow
+MANIFEST_OUTPUT?=$(FULL_OUTPUT_DIR)/manifest.json
+AMI_S3_DST=$(EXPORT_AMI_BUCKET)/$(ARTIFACTS_UPLOAD_PATH)
+EXPORT_AMI_DST=$(AMI_S3_DST)/$(EKSA_VERSION)/$(GIT_HASH)/
+LATEST_AMI_S3_URL=$(AMI_S3_DST)/$(LATEST)/snow-admin.raw
+
+BIN_FOLDER=bin
```

Comment:
I think this is fine, but for like kustomize we use in a few projects we end up just using the _output dir instead of introducing another temp dir, like bin.  In some cases we do `_output/dependencies'.  Im ok either way tho.